src/storage-types/src/connections.proto

```diff
@@ -75,6 +74,8 @@ message ProtoCsrConnectionHttpAuth {
 }
 
 message ProtoPostgresConnection {
+  reserved 9, 10, 11, 13;
+
```

Comment:
Do we need backwards-compat for this proto? (If it's just used for the cluster protocol, then no.)